libpromises/evalfunction.c

```diff
@@ -8722,6 +8722,28 @@ static FnCallResult FnCallCFEngineCallers(EvalContext *ctx, ARG_UNUSED const Pol
     return FnReturnContainerNoCopy(callers);
 }
 
+static FnCallResult FnCallString(EvalContext *ctx,
+                                 const Policy *policy,
+                                 const FnCall *fp,
+                                 const Rlist *finalargs)
+{
+    // Parser doesnt allow this to be true since STRING_ARGS has one argument
+    if(!finalargs) {
```

Comment:
```suggestion
    if(finalargs == NULL) {
```

https://github.com/cfengine/core/blob/master/CONTRIBUTING.md#safety--correctness